packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -614,18 +620,29 @@ export class GarbageCollector implements IGarbageCollector {
             if (this.testMode) {
                 this.deleteUnusedRoutes(gcResult.deletedNodeIds);
             }
+
+            if(this.writeDataAtRoot) {
```

Comment:
Does this cause extra work that we will do either way as part of summarization on container?
How expensive is it?
I assume we do it here because logically we included this info in gcStats and that's where we calculate it.

I worry a bit about it, especially in context of https://github.com/microsoft/FluidFramework/issues/9472 (and how large GC data can be).

Reply:
Ah, I think we generate it only once, would be nice to confirm